test/unit/MicroserviceErrorHandler.test.js

```diff
@@ -0,0 +1,90 @@
+const { expect } = require('chai');
+
+const MicroserviceErrorHandler = require('../../src/MicroserviceErrorHandler');
+
+describe('The MicroserviceErrorHandler class', function(){
+    it('can be instantiated with options containing the serviceName', function(){
+        const serviceName = 'unit-test-service';
+        const handler = new MicroserviceErrorHandler({ serviceName });
+
+        expect(handler).to.have.property('serviceName', serviceName);
+    });
+
+    it('can be instantiated with options containing the microservice config object', function(){
```

Comment:
What is the reason for the two interfaces to configure the name?

Reply:
Basically you could use the error handler without having our Microservice involved and then you probably don't want to add `{ "microservice": { "name": "blubb" }}` to your config.